netmanager/src/utils/setAuthToken.js

```diff
@@ -26,11 +26,10 @@ const setAuthToken = (token) => {
       return { ...config, params: { tenant, ...params } };
     });
   } else {
-    // Delete auth header
-    delete axios.defaults.headers.common['Authorization'];
-    localStorage.removeItem('mapStyle');
-    localStorage.removeItem('mapMode');
-    localStorage.removeItem('pollutant');
+    // Update auth header token to support data access for non-auth users
+    axios.defaults.headers.common[
+      'Authorization'
+    ] = `JWT ${process.env.REACT_APP_AUTHORIZATION_TOKEN}`;
   }
```

Comment:
On lines 30-32, as long as we don't keep this value in localStorage, the user will stay logged out because the current implementation checks if the token has been kept in localStorage to keep a user logged in.

Reply:
Hi @Codebmk , I am trying to appreciate this more.
You mean that we were only using the token for logged in users only? @123MwanjeMike

Reply:
There are two tokens. One is added to localStorage when the user logins. The second is just a placeholder so the map endpoints can return data without authorization(logically). To appreciate that properly, you may need to look through the entire setAuthToken.js file instead of just the changes to see what's going on.

Reply:
Thanks @Codebmk , just shared a calendar invite for a frontend workshop next Monday. I hope we can address more issues then.